### ISSUE TITLE
quickdop: command not found

### ISSUE THREAD

**Issue by aantron** [ocsigen/html_of_wiki#124]
See [this build log](https://travis-ci.org/github/ocsigen/lwt/builds/667571979#L560) in Lwt:

```
$ how-generate
quickdop: command not found
```

That log is from today. It was first observed on [23 March](https://travis-ci.org/github/ocsigen/lwt/builds/665716499). The last successful build of the docs, immediately before that, was on [14 March](https://travis-ci.org/github/ocsigen/lwt/builds/662403255). AFACT, nothing has changed on the Lwt end with respect to how the docs are built.

**Comment by aantron** [comment]
Ping. Would someone take a look at this?

**Comment by hhugo** [comment]
The issue seems to be that html_of_wiki fails to install.
```
[ERROR] The compilation of html_of_wiki failed at "/home/travis/.opam/opam-init/hooks/sandbox.sh build dune build -p html_of_wiki -j 1".
#=== ERROR while compiling html_of_wiki.2.0 ===================================#
# context              2.0.6 | linux/x86_64 | ocaml-base-compiler.4.06.1 | pinned(git+file:///home/travis/build/ocsigen/lwt/html_of_wiki#master#58154b0f)
# path                 ~/.opam/ocaml-base-compiler.4.06.1/.opam-switch/build/html_of_wiki.2.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p html_of_wiki -j 1
# exit-code            1
# env-file             ~/.opam/log/html_of_wiki-13096-1b1ef9.env
# output-file          ~/.opam/log/html_of_wiki-13096-1b1ef9.out
### output ###
# Error: This expression has type Html_types.phrasing Tyxml.Html.elt list
# [...]
#            | `Textarea
#            | `Time
#            | `U
#            | `Ul
#            | `Var
#            | `Wbr ] 
#        The first variant type does not allow tag(s)
#        `Address, `Article, `Aside, `Blockquote, `Details, `Div, `Dl,
#        `Fieldset, `Figure, `Footer, `Form, `H1, `H2, `H3, `H4, `H5, `H6,
#        `Header, `Hgroup, `Hr, `Main, `Menu, `Nav, `Ol, `P, `Pre, `Section,
#        `Style, `Table, `Ul
```

**Comment by hhugo** [comment]
The script in the CI should probably be fixed to fail at the first error

**Comment by hhugo** [comment]
Introduced by
```
commit 4588ee54ae9138ac160b3419d8edf07eed47ee34
Author: Drup <drupyog@zoho.com>
Date:   Thu Jan 23 15:14:11 2020 +0100

    Use "flow5" as content for th elements
    
    Fix #252
```

cc @Drup
